.github/workflows/release.yml

```diff
@@ -13,6 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: grafana/plugin-actions/build-plugin@release
+      - uses: grafana/plugin-actions/build-plugin@0315492b375faa7af0fa6c226ecb6c93c6fe9745
```

Comment:
I know that `release` tag is old but how did you choose this commit? Is it the latest one? If so, please add a comment saying that this is the latest commit on this day.

Reply:
```suggestion
      - uses: grafana/plugin-actions/build-plugin@0315492b375faa7af0fa6c226ecb6c93c6fe9745 # commit of 2024-07-08
```